src/pages/patch/patchRestartModal/TaskStatusCheckbox.tsx

```diff
@@ -2,30 +2,34 @@ import React from "react";
 import styled from "@emotion/styled";
 import Checkbox from "@leafygreen-ui/checkbox";
 import { mapVariantTaskStatusToColor, Square } from "components/StatusSquare";
-import { PatchBuildVariantTask } from "gql/generated/types";
-import { selectedStrings } from "hooks/usePatchStatusSelect";
 
 interface TaskStatusCheckboxProps {
-  task: PatchBuildVariantTask;
-  selectedTasks: selectedStrings;
-  toggleSelectedTask: (id: string) => void;
+  displayName: string;
+  status: string;
+  taskId: string;
+  checked: boolean;
+  style: React.CSSProperties;
```

Comment:
Would you mind adding a comment to this explaining why the style prop is here and that its used by react window. I can see us coming back to this in the future and wondering what it does.